### PR TITLE
Flyway alter table

### PR DESCRIPTION
**Pull request by TheSegi**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -25,31 +25,40 @@ dependencies {
...
+    implementation 'io.springfox:springfox-swagger-ui:2.7.0'
     implementation 'org.springframework.boot:spring-boot-starter-web:2.5.3'
     implementation 'org.springframework.boot:spring-boot-starter-data-jpa:2.5.3'
-    implementation 'org.springframework.boot:spring-boot-starter-data-rest:2.5.3'
+    //implementation 'org.springframework.boot:spring-boot-starter-data-rest:2.5.3'
     implementation 'org.springframework.boot:spring-boot-starter-validation:2.5.3'
     implementation 'org.springframework.boot:spring-boot-starter-security:2.5.3'
     implementation 'org.springframework.boot:spring-boot-starter-tomcat:2.5.3'
-    implementation 'mysql:mysql-connector-java:8.0.25'
+    implementation 'mysql:mysql-connector-java:8.0.26'
```
Comment (TheSegi) [inline build.gradle:37]: Enjoying github

File: build.gradle [hunk 2]
```diff
@@ -25,31 +25,40 @@ dependencies {
...
     implementation 'io.jsonwebtoken:jjwt:0.9.1'
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.2'
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.2'
     testImplementation 'org.mockito:mockito-core:3.11.2'
     implementation platform('com.amazonaws:aws-java-sdk-bom:1.11.1000')
     implementation 'com.amazonaws:aws-java-sdk-s3:1.12.44'
+    //flyway
+    implementation 'org.flywaydb:flyway-core:7.11.1'
```
Comment (TheSegi) [inline build.gradle:45]: Flyway is fun and easy

File: build.gradle [hunk 3]
```diff
@@ -25,31 +25,40 @@ dependencies {
...
 }
 
 test {
     useJUnitPlatform()
 }
 
 flyway {
-    url = 'jdbc:mysql://localhost:3306/mysqldemo?useSSL=false'
-    user = 'mysql'
```
Comment (TheSegi) [inline build.gradle:49]: my sql user id and password change.

**Review by TheSegi — COMMENTED** [review]
I like the comments

**Comment by TheSegi** [comment]
All good, ready to commit to Master.

**Comment by maltsevve** [comment]
Hi, TheSegi!
Thank you for your work!
I used Postman to send requests. I planned to add a collection of requests to the project later along with a few other small additions. But now I'm really interested in swagger. 
I'll double-check your code later and use it.

**Pull request closed without merging**